README.md

```diff
@@ -1,100 +1,155 @@
 # go-bt
+> The go-to Bitcoin Transaction (BT) GoLang library  
+
+[![Release](https://img.shields.io/github/release-pre/libsv/go-bt.svg?logo=github&style=flat&v=1)](https://github.com/libsv/go-bt/releases)
+[![Build Status](https://travis-ci.com/libsv/go-bt.svg?branch=master&v=1)](https://travis-ci.com/libsv/go-bt)
+[![Report](https://goreportcard.com/badge/github.com/libsv/go-bt?style=flat&v=1)](https://goreportcard.com/report/github.com/libsv/go-bt)
+[![codecov](https://codecov.io/gh/libsv/go-bt/branch/master/graph/badge.svg?v=1)](https://codecov.io/gh/libsv/go-bt)
+[![Go](https://img.shields.io/github/go-mod/go-version/libsv/go-bt?v=1)](https://golang.org/)
+[![Sponsor](https://img.shields.io/badge/sponsor-libsv-181717.svg?logo=github&style=flat&v=3)](https://github.com/sponsors/libsv)
+[![Donate](https://img.shields.io/badge/donate-bitcoin-ff9900.svg?logo=bitcoin&style=flat&v=3)](https://gobitcoinsv.com/#sponsor)
+
+<br/>
+
+## Table of Contents
+- [Installation](#installation)
+- [Documentation](#documentation)
+- [Examples & Tests](#examples--tests)
+- [Benchmarks](#benchmarks)
+- [Code Standards](#code-standards)
+- [Usage](#usage)
+- [Maintainers](#maintainers)
+- [Contributing](#contributing)
+- [License](#license)
+
+<br/>
 
-> The go-to Bitcoin Transaction (BT) GoLang library.  
+## Installation
 
-For more information around the technical aspects of Bitcoin, please see the updated [Bitcoin Wiki](https://wiki.bitcoinsv.io/index.php/Main_Page).
+**go-bt** requires a [supported release of Go](https://golang.org/doc/devel/release.html#policy).
+```shell script
+go get -u github.com/libsv/go-bt
+```
 
-## Documentation
+<br/>
 
-Check the [GoDoc](https://pkg.go.dev/mod/github.com/libsv/go-bt) documentation.
+## Documentation
+View the generated [documentation](https://pkg.go.dev/github.com/libsv/go-bt)
 
-## Installation
+[![GoDoc](https://godoc.org/github.com/libsv/go-bt?status.svg&style=flat)](https://pkg.go.dev/github.com/libsv/go-bt)
 
-**Install with [go](https://formulae.brew.sh/formula/go)**
+For more information around the technical aspects of Bitcoin, please see the updated [Bitcoin Wiki](https://wiki.bitcoinsv.io/index.php/Main_Page)
 
-```console
-go get github.com/libsv/go-bt
-```
+<br/>
 
-## Tests
+### Features
+- Full Featured Bitcoin Transactions
+- Auto-Fee Calculations for Change Address
+- Uses common type: [`mAPI.Fee`](https://github.com/bitcoin-sv/merchantapi-reference/blob/35ba2d395acc632eed1bd528ab524aeafad8bd40/utils/types.go#L32) for easy integration across projects
```

Comment:
@jadwahab please add more relevant features to mention

Reply:
302e9e8